common/core/desktop/include/keyman/keyboardprocessor_debug.h

```diff
@@ -61,10 +77,8 @@ typedef struct {
 typedef struct {
   uint32_t  type; // 32 bits is better optimized than 8 bits
   uint32_t flags;
-  union {
-    km_kbp_state_debug_key_info key_info;
-    km_kbp_state_debug_kmx_info kmx_info;
-  };
+  km_kbp_state_debug_key_info key_info;
+  km_kbp_state_debug_kmx_info kmx_info;
```

Comment:
While these are currently mutually exclusive, this may not remain this way, and it makes certain init simpler.